mne_bids/tests/test_meg_bids.py

```diff
@@ -51,9 +55,10 @@ def test_fif():
     raw.save(raw_fname2)
 
     raw_to_bids(subject_id=subject_id, run=run, task=task,
-                session_id=session_id, raw_fname=raw_fname2,
-                events_fname=events_fname, output_path=output_path,
+                session_id=session_id, raw_file=raw_fname2,
+                events_data=events_fname, output_path=output_path,
                 event_id=event_id, overwrite=True)
+    _strip_ieeg_from_json(output_path)
```

Comment:
Perhaps I am missing something but why do we need to do this if there is already an argument 'kind' which defaults to 'meg'? I think since the iEEG specification is not yet part of BIDS main specification there should be an option for the user to leave out the iEEG specific fields. What do you think @choldgraf ?